client/state/automated-transfer/middleware.js

```diff
@@ -1,29 +1,26 @@
 /**
  * Internal dependencies
  */
-import localforage from 'lib/localforage';
 import {
-	THEME_TRANSFER_STATUS_RECEIVE,
-	AUTOMATED_TRANSFER_STATUS_SET,
-	AUTOMATED_TRANSFER_ELIGIBILITY_UPDATE
+	THEME_TRANSFER_INITIATE_FAILURE,
+	THEME_TRANSFER_INITIATE_REQUEST,
+	THEME_TRANSFER_INITIATE_SUCCESS,
```

Comment:
Looks like `setAutomatedTransferStatus` isn't used yet, but I think we should bulletproof for it.